docs/DATABASE_SCHEMA.ja.md

```
@@ -1,61 +1,153 @@
-| skey        | ソートキー(集約種別名-集約IDの値部分-シーケンス番号)         | user-account-01H42K4ABWQ5V2XQEP3A48VE0Z-12345                                                                                                                                                                                                                                                                                                                                                                                                                        |    |
-| aid         | 集約ID                                  | user-account-01H42K4ABWQ5V2XQEP3A48VE0Z                                                                                                                                                                                                                                                                                                                                                                                                                              |    |
-| ser_nr      | シーケンス番号(開始番号は1)                       | 12345                                                                                                                                                                                                                                                                                                                                                                                                                                                              |    |
-| payload     | イベント内容                                | {"type":"Created","id":"01H42KBHCW1BZG504J4ZXKA2F2","aggregate_id":{"value":"01890535-c59c-72d5-08a8-dcea316374c8"},"seq_nr":1,"name":"test","members":{"members_ids_by_user_account_id":{"01H42KBHCWBDTZYQ7P78T8BTWX":"01H42KBHCWA8NE32M49YH544H1"},"members":{"01H42KBHCWA8NE32M49YH544H1":{"id":"01H42KBHCWA8NE32M49YH544H1","user_account_id":{"value":"01890535-c59c-5b75-ff5c-f63a3485eb9d"},"role":"Admin"}}},"occurred_at":"2023-06-29T03:32:37.404481Z"} |    |
-| occurred_at | 発生日時                                  | 2023-06-29T03:32:37.404481Z                                                                                                                                                                                                                                                                                                                                                                                                                                        |    |
+> v3 のレイアウトは v2 から変更されています（journal への `manifest` 列の追加、
+> `occurred_at` のナノ秒精度化、snapshot payload からのメタデータ注入の廃止、
+> current/履歴判別のキーへの移動）。v2 で書き込まれた行の読み取りはサポートしません。
+> 移行方針は [MIGRATION_GUIDE_v3.ja.md](MIGRATION_GUIDE_v3.ja.md) を参照してください。
 
-aidとseq_nrはGSIが適用されており、リプレイ時はこのインデックスを利用されます。
+### スナップショット保持のバックエンド間非対称
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)